### PR TITLE
✨ 82 - Add theming, repo components

### PR DESCRIPTION
**Pull request by anncatton**
Add theming, styled Arranger components from previous tickets (roadmap #70 and #68) to work with Next

### CODE REVIEW

File: components/pages/repository/RepoTable.tsx [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+                    //   <img
+                    //     alt="Download File"
+                    //     src={download}
+                    //     css={css`
+                    //       width: 16px;
+                    //       height: 16px;
+                    //     `}
+                    //   />
+                    // </span>
+                    // {/* </Tooltip> */}
```
Comment (hlminh2000) [inline components/pages/repository/RepoTable.tsx:240]: I'm guessing this is intentional, but is it because of an Emotional problem? lol

**Review by hlminh2000 — COMMENTED** [review]
small question :)

File: components/pages/repository/RepoTable.tsx (hunk 1 above)
Comment (anncatton) [reply components/pages/repository/RepoTable.tsx:240]: it's always an emotional problem lol...but for realsies, this column is gonna get removed in another ticket

Comment (anncatton) [reply components/pages/repository/RepoTable.tsx]: removed it!

**Review by hlminh2000 — APPROVED** [review]

**Pull request merged**